pandora-client-web/src/components/wardrobe/views/wardrobeLockLogic.tsx

```diff
@@ -236,6 +268,10 @@ export function WardrobeLockLogicUnlocked<TActionContext>({ lockLogic, ActionBut
 			{
 				lockLogic.lockSetup.timer ? (
 					<Column className='WardrobeLockTimer'>
+						<Row className='WardrobeInputRow'>
+							<label>Can be unlocked early by the locker</label>
+							<Checkbox checked={ timerAllowEarly } onChange={ setTimerAllowEarlyWithConfirm } />
```

Comment:
The text feels quite hard to understand.
Suggested alternative (as only the person locking it can control it anyway, as it isn't persistent while unlocked):
> The lock can be unlocked by you even the before timer runs out